src/harmony/Badge/Badge.tsx

```diff
@@ -77,21 +77,21 @@ export const Badge = forwardRef(
                 className={cn(s.Badge, className, view ? viewMap[view] : '', sizeMap[size])}
             >
                 {nullable(iconLeft, (icon) => (
-                    <span className={cn(s.BadgeIcon, { [s.BadgeIconLeft]: text })}>{icon}</span>
+                    <div className={cn(s.BadgeIcon, { [s.BadgeIconLeft]: text })}>{icon}</div>
```

Comment:
Why did it cause hydration issues?

Reply:
Next doesn't like `div` inside `span`
![image](https://github.com/taskany-inc/bricks/assets/9020793/51edd671-d670-42b1-98bd-84da450e13dd)